proto/mls/message_contents/credential.proto

```diff
@@ -0,0 +1,31 @@
+// Credentials and revocations
```

Comment:
I _believe_ we can leave this deleted too

Reply:
Gone